aip/general/0134.md

```diff
@@ -160,7 +160,7 @@ wiped out data because the previous version did not know about it.
 
 Some resources take longer to update a resource than is reasonable for a
 regular API request. In this situation, the API **should** use a long-running
-operation (AIP-151) instead:
+operation [AIP-151][] instead:
```

Comment:
```suggestion
operation ([AIP-151][]) instead:
```

We still want it to be parenthetical as it is a reference to the AIP to click through for more details.